docker/CMSRucioClient/scripts/cmslinks.py

```diff
@@ -76,17 +76,14 @@ def _get_rselist(self, rselist=None):
                         pnn = site.get('site')
                         break
 
-            try:
-                self.rselist.append({
-                    'rse': rse,
-                    'pnn': pnn,
-                    'type': attrs['cms_type'],
-                    'country': attrs['country'],
-                    'region': attrs.get('region', None)
-                })
-            except KeyError:
-                logging.warning('No expected attributes for RSE %s. Skipping',
-                                rse)
+            self.rselist.append({
+                'rse': rse,
+                'pnn': pnn,
+                'type': attrs.get('cms_type'),
+                'country': attrs.get('country'),
+                'region': attrs.get('region', None)
```

Comment:
I meant, 
`'region': attrs.get('region', None)` -> `'region': attrs.get('region'`
This would fail only for the misconfigured site with key error and log a warning.